wagtailio/settings/base.py

```diff
@@ -151,6 +151,10 @@
     MIDDLEWARE.insert(0, "baipw.middleware.BasicAuthIPWhitelistMiddleware")
     BASIC_AUTH_LOGIN = "wagtailio"
     BASIC_AUTH_PASSWORD = "showmewagtailio"
```

Comment:
while here, it would be great to update this bit to properly get the credentials from the environment variables, rather than hardcoded values

Reply:
Thanks @zerolab, updated in 6ae8d4a3